lemmy_db/src/activity.rs

```diff
@@ -62,7 +62,8 @@ pub fn do_insert_activity<T>(
 where
   T: Serialize + Debug,
 {
-  debug!("inserting activity for user {}, data {:?}", user_id, &data);
+  debug!("inserting activity for user {}: ", user_id);
+  debug!("{}", serde_json::to_string_pretty(&data)?);
```

Comment:
This is probably good to leave in here, its been helpful for debugging.

Reply:
This would be good for verbose logs, but that doesnt seem to exist. And if we had actual HTTP endpoints for activities we could just get them from there. 

So for now this is probably the best we can do. Maybe change the shorter ones to `info!` though?

Reply:
hrm.... I'm up for any log changes, I think it goes error, info, debug. But we can do that in a different issue.